src/Sylius/Component/Core/Storage/CartStorageInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Storage;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+
+interface CartStorageInterface
+{
+    /**
+     * @param ChannelInterface $channel
+     *
+     * @return bool
+     */
+    public function hasCartIdForChannel(ChannelInterface $channel): bool;
+
+    /**
+     * @param ChannelInterface $channel
+     *
+     * @return mixed
```

Comment:
We need to make it more stricter, either keep it as `getCartIdForChannel` and define an exception to throw when cart is not found or rename it to `findCartIdForChannel` and return null when there's no cart for a given channel.